.github/workflows/deploy.yml

```
@@ -0,0 +1,77 @@
+            echo "${{ secrets.ENV_PROD }}" > .env
+
+            # 컨테이너 중지, 삭제 및 이미지 pull
+            docker stop mindwalk-server || true
+            docker rm mindwalk-server || true
+            docker pull ${{ steps.login-ecr.outputs.registry }}/mindwalk-server:latest
+
+            # 컨테이너 실행
+            docker run -d --name mindwalk-server -p 8080:8080 \
+              --env-file .env \
+              ${{ steps.login-ecr.outputs.registry }}/mindwalk-server:latest
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)